File: app/src/main/java/com/example/getfood/ui/orderlist/OrderListMvpView.java
package com.example.getfood.ui.orderlist;

import android.content.Context;

import com.example.getfood.ui.base.BaseView;
import com.fazemeright.canteen_app_models.models.FullOrder;

import java.util.ArrayList;

public interface OrderListMvpView extends BaseView {
    void bindListAdapter(ArrayList<FullOrder> orderListItems);

    Context getContext();

    void onRollNumberNull();
}

File: app/src/main/java/com/example/getfood/ui/orderlist/OrderListMvpView.kt
package com.example.getfood.ui.orderlist

import com.example.canteen_app_models.models.FullOrder
import com.example.getfood.ui.base.BaseView
import java.util.*

interface OrderListMvpView : BaseView {
    fun bindListAdapter(orderListItems: ArrayList<FullOrder?>?)

    fun onRollNumberNull()
}